runtime/src/iree/hal/drivers/cuda/cuda_device.c

```diff
@@ -294,6 +301,50 @@ IREE_API_EXPORT iree_status_t iree_hal_cuda_nccl_get_unique_id(
                                                        out_id);
 }
 
+IREE_API_EXPORT iree_status_t iree_hal_cuda_nccl_query_group_params(
+    void* self, iree_hal_device_t* device,
+    iree_hal_queue_affinity_t queue_affinity, iree_byte_span_t id_storage,
+    iree_hal_channel_params_t* params) {
```

Comment:
```suggestion
    iree_hal_channel_params_t* out_params) {
```
(here and elsewhere)
convention to indicate that an argument is an output prefix with `out_` - this lets the caller know that it's only written and the contents are not expected to be populated prior to calling